### ISSUE TITLE
[Feature] Context aware right click menus for lightning-related actions

### ISSUE THREAD

**Issue by reneaaron** [getAlby/lightning-browser-extension#3015]
### Feature description

![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/eac32241-7b77-4061-913e-7b67856e8885)

### Describe the solution

Whenever a user selects some text Alby could check if the selected text can be handled (e.g. invoices, LNURL-withdraw, lightning addresses, etc) and display context menus. 

### Are you working on this?

None

**Comment by stackingsaunter** [comment]
Shall we add npubs there? User could choose a client in the settings?

**Comment by reneaaron** [comment]
No, let's keep that separate. It's 2 completely unrelated things that can be done separately.
